src/main/java/com/google/sps/servlets/SearchServlet.java

```diff
@@ -32,6 +34,22 @@
 /** Servlet that returns a list of tutors for a searched topic. */
 @WebServlet("/search")
 public class SearchServlet extends HttpServlet {
+    private SearchDatastoreService datastore;
+
+    public SearchServlet() {}
```

Comment:
Why is this one empty? Shouldn't just the one below be kept instead?

Reply:
Good question! I should probably add a comment to clarify this. Because I specified a constructor with a parameter (the testing one), the default empty constructor does not work anymore so I have to explicitly create it. We need the default one for deployment because the servlet is created without parameters.

Reply:
Oh I see! Sounds good :)